packages/user-interface/src/index.ts

```diff
@@ -9,6 +9,7 @@ export { default as PageGrid } from "./components/PageGrid";
 export { default as PortalLink } from "./components/PortalLink";
 export { default as TableList } from "./components/TableList";
 export { default as Table } from "./components/Table";
+export * from "./components/Table";
```

Comment:
Moeten we dan niet gewoon overal ster van maken? weet niet meer waarom we default hebben gebruikt?

Reply:
@Borai Ik zie dat op veel plekken alleen de default wordt geëxporteerd. Als we overal ster van maken in plaats van {default as ... }, dan wordt er niks meer geëxporteerd. (Bij ster wordt namelijk de default niet geëxporteerd.)

Misschien moeten we later een keer kijken of er een betere oplossing is.

Reply:
Kunnen ook default + * overal doen dan ben je er ook toch?